packages/component-library/src/components/plmg-tooltip/test/plmg-tooltip.e2e.ts

```diff
@@ -0,0 +1,48 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { AxePuppeteer } from '@axe-core/puppeteer';
+import { Page } from 'puppeteer';
+
+describe('plmg-tooltip', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<plmg-tooltip></plmg-tooltip>');
+
+    const element = await page.find('plmg-tooltip');
+
+    expect(element).toHaveClass('hydrated');
+  });
+
+  describe('all possible variations', () => {
+    it('are accessible', async () => {
+      const page = await newE2EPage();
+
+      const arrowSides = ['none', 'left', 'right', 'top', 'bottom'];
+      const arrowPositions = ['start', 'middle', 'end'];
+      const bgColors = ['neutral', 'primary'];
+
+      let htmlContent = '';
+      bgColors.forEach((bgcolor) => {
+        arrowSides.forEach((arrowSide) => {
+          arrowPositions.forEach((arrowPosition) => {
+            htmlContent += `
+            <plmg-tooltip bg-color="${bgcolor} arrow-side="${arrowSide} arrow-position="${arrowPosition}">
```

Comment:
we probably need to add `force-visible="true"` otherwise nothing is rendered on the page